### PR TITLE
DEI-515: Test the ts-deploy-task-script-action Action

### PR DESCRIPTION
**Pull request by VladimirShteyn**
See here: https://github.com/tetrascience/ts-deploy-task-script-action/pull/41

- [ ] Before merge, update to point to deploy action `@v0.12.1`.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -355,3 +355,6 @@ pip-selfcheck.json
 # Support for Project snippet scope
 
 # End of https://www.toptal.com/developers/gitignore/api/python,pycharm+all,emacs,visualstudiocode,virtualenv,vim
+
+# Mac
+.DS_Store
```
Comment (mlipschultz) [inline .gitignore:360]: As a non-Mac user, it delights me to see this!

File: .github/workflows/publish.yml [hunk 2]
```diff
@@ -79,7 +79,7 @@ jobs:
           JFROG_ARTIFACTORY_USER: ${{ secrets.JFROG_ARTIFACTORY_USER }}
           JFROG_ARTIFACTORY_PASSWORD: ${{ secrets.JFROG_ARTIFACTORY_PASSWORD }}
         # NOTE: update reference here to use a different version of the action
-        uses: tetrascience/ts-deploy-task-script-action@v0.11.0
+        uses: tetrascience/ts-deploy-task-script-action@DEI-515-bug-similar-name-artifacts
```
Comment (mlipschultz) [inline .github/workflows/publish.yml:82]: These will need to be changed to `0.12.1` before merging.  I wonder why it wasn't ever bumped to use the latest (`0.12.0`).

**Review by mlipschultz — APPROVED** [review]

File: .github/workflows/publish.yml (hunk 2 above)
Comment (VladimirShteyn) [reply .github/workflows/publish.yml]: I was wondering that myself!

Comment (VladimirShteyn) [reply .github/workflows/publish.yml]: Done.

File: .gitignore (hunk 1 above)
Comment (VladimirShteyn) [reply .gitignore:360]: 😸

**Pull request merged**